src/content/en/tools/workbox/get-started/webpack.md

```diff
@@ -140,37 +147,92 @@ that they had an internet connection.
    uses for any matching URL. See [The Offline Cookbook][cookbook] for more
    on caching strategies.
 
-    <pre class="prettyprint">new workboxPlugin({
-      globDirectory: dist,
-      globPatterns: ['**/*.{html,js}'],
-      swDest: path.join(dist, 'sw.js'),
+    <pre class="prettyprint">new workboxPlugin.GenerateSW({
+      swDest: 'sw.js',
       clientsClaim: true,
       skipWaiting: true,
-      <strong>runtimeCaching: [
-        {
-          urlPattern: new RegExp('https://hacker-news\.firebaseio\.com'),
-          handler: 'staleWhileRevalidate'
-        }
-      ]</strong>
+      <strong>runtimeCaching: [{
+        urlPattern: new RegExp('https://hacker-news.firebaseio.com'),
+        handler: 'staleWhileRevalidate'
+      }]</strong>
     })</pre>
 
 [cookbook]: /web/fundamentals/instant-and-offline/offline-cookbook/
 
 <<_shared/try-complete.md>>
 
-<<_shared/create.md>>
+## Step 6: Create your own service worker {: #inject }
```

Comment:
I needed to manually copy over the `create.md` content and then lightly edit it, because there are some specific differences vs. what's in the shared doc when it comes to the v3 webpack plugin.